vue-frontend/src/components/portfolio/LandingSection.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div>
+    <div>{{ jsonData.title }}</div>
+    <div>{{ jsonData.subTitle }}</div>
+    <img :src="jsonData.backgroundImage" />
+  </div>
+</template>
+
+<script>
+import luxeradioJsonData from "@/luxeradio-data.json";
```

Comment:
JSON data is dynamic for the landing section, we can not directly assign luxeradio data to this section

Reply:
I know, that's why add that if condition in `mounted hook` and assign only `luxeradioJsonData.details.landingSection`